README.md

```diff
@@ -54,25 +54,28 @@ See the help text for more detailed usage instructions.
 
 ## Semantic Versioning
 
-`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The basic comparison operators are:
+`dep ensure` a uses an external [semver library](https://github.com/Masterminds/semver) to interpret the version constraints you specify in the manifest. The comparison operators are:
 
 * `=`: equal (aliased to no operator)
 * `!=`: not equal
 * `>`: greater than
 * `<`: less than
 * `>=`: greater than or equal to
 * `<=`: less than or equal to
+* `<=`: less than or equal to
+* `-`: literal range. Eg: 1.2.4 - 1.3
+* `~`: minor range. Eg: ~1.2.3 is equivalent to >= 1.2.3, < 1.3.0
+* `^`: major range. Eg: ^1.2.3 is equivalent to >= 1.2.3, < 2.0.0
+* `[xX*]`: wildcard. Eg: >=1.2.x
```

Comment:
Let's change the example to `1.2.x is equivalent to >= 1.2.0, < 1.3.0` so that it's a bit more clear how the wildcard works all by itself.